src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -60,8 +60,7 @@ def create_resource(
             f"Attempting to create resource {resource.res_id} (label: {resource.label}, iri: {resource.iri})..."
         )
         resource_dict = self._make_resource_with_values(resource, bitstream_information)
-        headers = {"X-Asset-Ingested": "true"} if self.media_previously_ingested else None
-        res = self.con.post(route="/v2/resources", data=resource_dict, headers=headers)
+        res = self.con.post(route="/v2/resources", data=resource_dict, headers={"X-Asset-Ingested": "true"})
```

Comment:
So it is ok, to give this header with all requests even if there are no assets that were ingested?

Reply:
Yes.